src/gc/gcinterface.h

```diff
@@ -174,6 +183,10 @@ enum end_no_gc_region_status
 
 typedef BOOL (* walk_fn)(Object*, void*);
 typedef void (* gen_walk_fn)(void* context, int generation, uint8_t* range_start, uint8_t* range_end, uint8_t* range_reserved);
+typedef void (* record_surv_fn)(uint8_t* begin, uint8_t* end, ptrdiff_t reloc, size_t context, BOOL compacting_p, BOOL bgc_p);
+typedef void (* fq_walk_fn)(BOOL, void*);
+typedef void (* fq_scan_fn)(Object** ppObject, ScanContext *pSC, uint32_t dwFlags);
+typedef void (* handle_scan_fn)(Object** pRef, Object* pSec, uint32_t flags, ScanContext* context, BOOL isDependent);
```

Comment:
they are different contexts. I am passing ScanContext* where it should be a ScanContext.

---
In reply to: [89182964](https://github.com/dotnet/coreclr/pull/8199#discussion_r89182964) [](ancestors = 89182964)